### PR TITLE
apply feedback from the CR version

### PR DESCRIPTION
**Pull request by ashimura**
I've  checked the diffs between the [master branch version](https://w3c.github.io/wot-architecture/)  and the [CR published version](https://www.w3.org/TR/2019/CR-wot-architecture-20190516/Overview.html) and generated a pullrequest to apply the changes.

Note that some of the edits can't be applied to the master branch version due to ReSpec's restriction, we still need some minor edit when we publish a Proposed Recommendation.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -273,6 +275,32 @@
...
+    preliminary implementation report</a>.
+    </p>
+
+    <p>This document was published by the <a href=
+    "https://www.w3.org/WoT/WG/">Web of Things Working Group</a> as
+      an Editor's Draft.
+     This document is intended to become a <abbr title="World Wide Web Consortium">W3C</abbr> Recommendation.</p>
+
+    <p>Please see the Working Group's
+    <a href="https://w3c.github.io/vc-test-suite/implementations/">
```
Comment (k-toumura) [inline index.html]: this should be 'https://w3c.github.io/wot-thing-description/testing/report.html'.

**Comment by mkovatsc** [comment]
> https://w3c.github.io/vc-test-suite/implementations/

That is something that should probably also be fixed in the CR at https://www.w3.org/TR/2019/CR-wot-architecture-20190516/

**Comment by mlagally** [comment]
@ashimura I looked at the diff and am not sure I understand your comment about the minor edit for respec. Do you think we can merge this PR after review?

**Comment by ashimura** [comment]
That's true... I'll talk with the Webmaster about that.

File: index.html (hunk 1 above)
Comment (ashimura) [reply index.html]: fixed.

Comment (ashimura) [reply index.html]: fixed.

**Comment by ashimura** [comment]
@mkovatsc no problem with the CR now :)

File: index.html [hunk 2]
```diff
@@ -612,7 +640,7 @@ <h3>Consumer</h3>
                     and other UX devices.</p>
                 <figure id="smart-home">
                     <img src="images/wot-use-cases/smart-home.png"
-                        class="wot-arch-diagram"/>
+                        class="wot-arch-diagram"/ alt="smart home use case" />
```
Comment (mkovatsc) [inline index.html]: The slash before `alt` has to go.

File: index.html [hunk 3]
```diff
@@ -3238,7 +3266,7 @@ <h3>Connection Through Proxy Synchronization</h3>
                 </p>
                 <figure id="deployment-service-sync-proxy">
                     <img src="images/deployments/arch-deploy-service-sync-intermediary.png"
-                         class="wot-arch-diagram" />
+                         class="wot-arch-diagram" alt="service sync proxy" />
```
Comment (mkovatsc) [inline index.html]: might want to call it "intermediary" also in the alt text.

File: index.html (hunk 2 above)
Comment (ashimura) [reply index.html]: fixed.

File: index.html (hunk 3 above)
Comment (ashimura) [reply index.html]: In that case, maybe it would be better to use "deployment-service-sync-intermediary" for id as well. What do you think?

Comment (mkovatsc) [reply index.html]: Yes, sounds good. Need to double check if it is referenced from somewhere and fix there as well...

File: index.html (hunk 2 above)
Comment (mkovatsc) [reply index.html]: Good, this was the main issue -- approved changes

File: index.html (hunk 3 above)
Comment (ashimura) [reply index.html]: OK. I've checked the index.html file again for "proxy" and converted it to "intermediary" if appropriate, e.g., the PNG file's name is *-intermediary.png".

Let's review the changes during the Architecture call tomorrow on May 23.

**Pull request merged**